plugins/ScheduledReports/templates/index.twig

```diff
@@ -32,6 +32,7 @@
 </div>
 
 <script type="text/javascript">
+    var timeZoneDifference = {{ timeZoneDifference }} / 3600;
```

Comment:
Did you see Adam's comment in previous PR? Do you think it would be a good idea to move this to separate function?